### PR TITLE
Fix/security finding

### PR DESCRIPTION
**Pull request by thackerronak**
## Description
Spring boot and cloud lib version was updated to mitigate https://github.com/advisories/GHSA-f3jh-qvm4-mg39

Note: 
Below are two dependencies showing restricted which need to be fixed in ssi-lib. 
maven/mavencentral/org.jetbrains.kotlin/kotlin-stdlib-jdk7/1.9.23, None, restricted, #14188
maven/mavencentral/org.jetbrains.kotlin/kotlin-stdlib-jdk8/1.9.23, None, restricted, #14185

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by borisrizov-zf** [comment]
@thackerronak Hi, we cannot merge a PR which includes "restricted" dependencies. You also mentioned that this has to do with the SSI lib. Can you elaborate?

**Comment by thackerronak** [comment]
> @thackerronak Hi, we cannot merge a PR which includes "restricted" dependencies. You also mentioned that this has to do with the SSI lib. Can you elaborate?

@borisrizov-zf I have updated the DEPENDENCIES file and found that all dependencies are showing approved.

**Review by nitin-vavdiya — APPROVED** [review]

File: gradle.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
-springCloudVersion=2022.0.3
```
Comment (nitin-vavdiya) [inline gradle.properties:1]: Please add license header

**Review by nitin-vavdiya — CHANGES_REQUESTED** [review]

**Review by nitin-vavdiya — APPROVED** [review]

**Comment by nitin-vavdiya** [comment]
@thackerronak  Please rebase and re-generate DEPENDENCIES

**Comment by nitin-vavdiya** [comment]
@thackerronak  please rebase and  re-generate DEPENDENCIES

**Pull request closed without merging**